ResearchKit/Common/ORKAnswerFormat.h

```diff
@@ -246,8 +246,11 @@ ORK_CLASS_AVAILABLE
                                                 calendar:(nullable NSCalendar *)calendar;
 
 + (ORKTextAnswerFormat *)textAnswerFormat;
+
 + (ORKTextAnswerFormat *)textAnswerFormatWithMaximumLength:(NSInteger)maximumLength;
 
++ (ORKTextAnswerFormat *)textAnswerFormatWithValidationExpression:(NSString *)expression validInputDescription:(NSString *)description;
```

Comment:
Update the initializer to reflect the refactored property name `invalidMessage`.
Maybe change it to `invalidMessage:(NSString *)invalidMessage`?